src/FixedDataTableCellDefaultDeprecated.js

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
This file is just the old `FixedDataTableCellDefault`. 
There's no change besides a comment explaining this component is deprecated.